pygyw/layout/fonts.py

```diff
@@ -57,32 +50,14 @@ def to_json(self) -> Dict[str, Any]:
         """Return a JSON-serializable dictionary of the object."""
 
         return {
-            "title": self.index,
+            "title": self.filename,
         }
 
 
 class GYWFonts:
-    """
-    Active fonts on aRdent smart glasses.
-
-    Attributes:
-        SMALL: A small font
-        MEDIUM: A medium font
-        LARGE: A large font
-        HUGE: A huge font
-        values: List containing every fonts available on aRdent smart glasses.
-    """
-
-    SMALL = GYWFont(
-        name="Small", index=0, prefix="a10", size=18, width=10, height=25)
-
-    MEDIUM = GYWFont(
-        name="Medium", index=1, prefix="b14", size=24, width=14, height=33, bold=True)
-
-    LARGE = GYWFont(
-        name="Large", index=2, prefix="a16", size=28, width=16, height=39)
+    """Active fonts on aRdent smart glasses."""
 
-    HUGE = GYWFont(
-        name="Huge", index=3, prefix="b28", size=48, width=28, height=67, bold=True)
+    ROBOTO_MONO = GYWFont(name="Roboto Mono", filename="robmn")
+    ROBOTO_MONO_BOLD = GYWFont(name="Roboto Mono Bold", filename="robmb", bold=True)
 
-    values = [SMALL, MEDIUM, LARGE, HUGE]
+    values = [ROBOTO_MONO, ROBOTO_MONO_BOLD]
```

Comment:
Why don't you add `ITALIC` and `BOLD_ITALIC` ?

Reply:
I didn't know if we're adding those values now or later as a new feature. I'll add them now then.

Reply:
What filename do you prefer for the bold italic? We only have one letter available for the style (`robmb` for bold), not 2.